Display/features/gets_sensor_data.feature

```diff
@@ -0,0 +1,15 @@
+Feature: gets data from sensors
+
+    Scenario: connects to the sensors that are connected
+        Given some sensors are physically connected to the Pressure Sensor and Airflow Sensor PI Hat
+        Then the software will determine which tubes have both pressure and airflow sensing
+        Then the software will initialize and calibrate the sensors
+        Then the software will return the state of the sensor to the user.
+
+    Scenario: there are not enough sensors
+        Given there are no tubes with both pressure and airflow sensing connected
+        Then the software will raise an error
+
+    Scenario: there are multiple airflow sensors on a single tube
+        Given more than one airflow sensor is connected to a single tube
```

Comment:
- Given sensor A on a tube
- and less functional sensor B on a tube
- When data is read
- then the data from A will be used

Something like this means you can have lots of different sensor "givens" and reuse them, and have a "when" that is "reading the data" that you reuse between tests that goes through all the initialized sensors in the context.

You will need to define what "worse" is here,